airflow/providers/postgres/hooks/postgres.py

```diff
@@ -192,7 +192,7 @@ def get_iam_token(self, conn: Connection) -> Tuple[str, str, int]:
             # Pull the custer-identifier from the beginning of the Redshift URL
             # ex. my-cluster.ccdre4hpd39h.us-east-1.redshift.amazonaws.com returns my-cluster
             cluster_identifier = conn.extra_dejson.get('cluster-identifier', conn.host.split('.')[0])
-            session, endpoint_url = aws_hook._get_credentials()
+            session, endpoint_url = aws_hook._get_credentials(region_name=None)
```

Comment:
was just surprised to see aws_hook in postgres, Im sure its done for a reson.

Reply:
Yep. This is to support RedShift.

Reply:
Wait. I think this might be not backwards compatible....

Reply:
No. It's OK.